src/store/reducers/Auth.ts

```diff
@@ -0,0 +1,23 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
```

Comment:
Create an `auth` folder and put this file as `store/auth/authSlice.ts` will be better.
Sometimes in this folder we will need more than a `.*Slice.ts` file but also an other file to store type definitions.